source/Calamari.AzureWebApp/Util/AzureWebAppHelper.cs

```diff
@@ -31,5 +36,21 @@ public static AzureTargetSite GetAzureTargetSite(string siteAndMaybeSlotName, st
             targetSite.Slot = slotName;
             return targetSite;
         }
+       
+        public static async Task<bool> GetBasicPublishingCredentialsPoliciesAsync(string resourceManagementEndpointBaseUri , string subscriptionId, string resourceGroupName, string webAppName, string accessToken)
+        {
+            var url = $"{resourceManagementEndpointBaseUri}/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/sites/{webAppName}/basicPublishingCredentialsPolicies/scm?api-version=2021-01-15";
```

Comment:
This might not matter or be be handled already but the default value includes a forward slash already so we end up with `https://management.azure.com//subscirptions` here. If this fails any tests I think we may need to check for a trailing '/' and trim if it's there.